src/core_plugins/console/api_server/spec/overrides/indices.create.json

```diff
@@ -1,8 +1,5 @@
 {
-  "{index}": {
-    "patterns": [
-      "{index}"
-    ],
+  "indices.create": {
     "data_autocomplete_rules": {
       "mappings": {
         "__scope_link": "_put_mapping"
```

Comment:
looks like `_put_mapping` used to be an endpoint defined here: https://github.com/elastic/kibana/pull/18930/files#diff-5a9db413dff420db68b109840db90543L177. Do you think we can check that all of the `__scope_link` references line up with a script or something?

Reply:
good idea